public/app/features/provisioning/RepositoryStatusPage.tsx

```diff
@@ -112,7 +121,15 @@ export default function RepositoryStatusPage() {
                   {tab === TabSelection.Overview && <RepositoryOverview repo={data} />}
                   {tab === TabSelection.Resources && <RepositoryResources repo={data} />}
                   {tab === TabSelection.Files && <FilesView repo={data} />}
-                  {tab === TabSelection.Export && <ExportToRepository repo={data} />}
+                  {tab === TabSelection.Export && (
```

Comment:
We will need to change this one to a button if we merge this one before my other PR.